ui/src/Upload.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { useReducer } from 'react';
+import { AppDispatchContext, AppDispatchReducer, initialState } from './reducers/AppDispatchReducer';
+import './Upload.css';
+import MemeUpload from './MemeUpload';
+import ImageUpload from './ImageUpload';
+
+function Upload(){
```

Comment:
Should the name of this be more like `CreateMeme` rather than `Upload`? Curious on thoughts